drivers/cpufreq/cpufreq_performance.c

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ *  linux/drivers/cpufreq/cpufreq_performance.c
+ *
+ *  Copyright (C) 2002 - 2003 Dominik Brodowski <linux@brodo.de>
+ */
+
```

Comment:
@GUIDINGLI I think we cannot use this because GPL license is incompatible with Apache License.